client/src/components/ReviewList.jsx

```diff
@@ -15,11 +15,10 @@ class Reviews extends React.Component {
 
     componentDidMount() {
         const context = this;
-        var url = window.location.href.split('/').pop();
+        let url = window.location.href.split('/').pop();
```

Comment:
Problem: Getting the id from the url makes it difficult to reuse this component in other scenarios.

Fix: Have the ReviewList component accept the id as a prop.